db/schemaMigrations.ts

```diff
@@ -27,9 +28,10 @@ export const getMigrationVersion = (fileName: string): number | null => {
 const migrationInsert = (
   migrationNumber: number,
   id?: string
-) => `INSERT INTO "public"."migrations" ("id", "number", "version", "executed", "processed_block", "created_at", "updated_at")
-VALUES ('${id || migrationNumber}', ${migrationNumber}, '${latestVersion}', 0, 0, now(), now())
-ON CONFLICT(id) DO UPDATE SET "updated_at" = now();`;
```

Comment:
With historical state enabled unique constraints don't really work. The "executed" entry will get added in.

I wasn't able to get any duplicate migrations to appear when testing though.